Add ob-pic recipe

ddoherty03 opened the pull request:
### Brief summary of what the package does

Adds support for pic language to org babel.

### Direct link to the package repository

https://github.com/your/ddoherty03/ob-pic

### Your association with the package

Author and Maintainer

### Relevant communications with the upstream package maintainer

** None needed **

### Checklist

<!-- Please confirm by replacing `[]` with `[x]`: -->

- [X] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [X] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [X] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [X] My elisp byte-compiles cleanly
- [X] I've used `M-x checkdoc` to check the package's documentation strings
- [X] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

<!-- After submitting, please fix any problems the CI reports. -->

riscy commented:
Thanks for this.  Quick first pass -

- ob-pic.el needs *formal* license boilerplate and/or an [SPDX-License-Identifier](https://spdx.dev/ids/)
- In your `;;; Commentary` you point users to look at https://github.com/ddoherty03/ob-pic but I think a good Commentary would include the basic information about the kind of source block this works on, with an example.  (E.g. imagine the user doesn't have network access at the time, or they're browsing MELPA.org and they'll be inconvenienced by having to copy-paste your URL and open a new browser window to verify what this is)

riscy commented:
Friendly ping. :)

ddoherty03 commented:
Chris,

I thought I addressed your comments in my commit on December 9, 2024.

Am I still missing something?

Cheers,

====================================================
Daniel E. Doherty
D.E. Doherty Law Office, LLC
7300 W. 110th Street, Suite 930
Overland Park, KS 66210
913.338.7182 (Phone)
913.338.7164 (FAX)
***@***.***




On Sun, Feb 9, 2025 at 2:23 PM Chris Rayner ***@***.***>
wrote:

> Friendly ping. :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/melpa/melpa/pull/9273#issuecomment-2646570858>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADQO3ETMQUYSYIKCOFQ5CD2O62NNAVCNFSM6AAAAABTGIQGCWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNBWGU3TAOBVHA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

riscy commented:
> I thought I addressed your comments in my commit on December 9, 2024.

Yes, I see now.  (It's buried in the second-last paragraph of CONTRIBUTING.org, but it's best if you let us know explicitly here when you make a change.)

Thanks for accommodating.  I'll merge this now - and welcome to MELPA. :)

ddoherty03 commented:
Thanks,Chris.

====================================================
Daniel E. Doherty
D.E. Doherty Law Office, LLC
7300 W. 110th Street, Suite 930
Overland Park, KS 66210
913.338.7182 (Phone)
913.338.7164 (FAX)
***@***.***




On Sun, Feb 16, 2025 at 12:03 PM Chris Rayner ***@***.***>
wrote:

> Merged #9273 <https://github.com/melpa/melpa/pull/9273> into master.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/melpa/melpa/pull/9273#event-16324846490>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADQO3HBY4FXGYMGFOVUOHD2QDHHPAVCNFSM6AAAAABTGIQGCWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJWGMZDIOBUGY2DSMA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
